build.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-  kotlinVersion = '1.2.50'
+  kotlinVersion = '1.2.51'
```

Comment:
My Kotlin plugin complained about this but if you want i can take it out of the review.